flows/common/common.go

```diff
@@ -3,12 +3,152 @@ package common
 
 import (
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
 	"github.com/open-traffic-generator/snappi/gosnappi/otg"
 	"github.com/openconfig/magna/lwotg"
+	"k8s.io/klog"
 )
 
+var (
+	// timeout specifies how long to wait for a PCAP handle.
+	pcapTimeout = 30 * time.Second
+	// packetBytes is the number of bytes to read from an input packet.
+	packetBytes int = 100
+)
+
+// hdrsFunc is a function which specifies which packet headers to expect.
```

Comment:
Should we clarify that this is used to define what packet headers to create as well as expect? (Since you use `fn` on line 35 to specify what the sent packets are going to look like.)